task/02-numbers-tasks.js

```diff
@@ -111,7 +111,7 @@ function getLinearEquationRoot(a, b) {
  *   (0,1) (1,2)     => 0
  */
 function getAngleBetweenVectors(x1, y1, x2, y2) {
-    throw new Error('Not implemented');
+    return Math.acos((x1*x2+y1*y2)/(Math.sqrt((Math.pow(x1,2)+Math.pow(y1,2)))*Math.sqrt(Math.pow(x2,2)+Math.pow(y2,2))));
```

Comment:
This is fine, although not readable. You could use getDistanceBetweenPoints function here